### PR TITLE
Remove usage of UBER_CONFIG_DIR

### LINKED ISSUE #11: Remove logic about UBER_CONFIG_DIR

### PR DESCRIPTION
**Pull request by yiranwang52**
fix #11

### CODE REVIEW

File: utils/configutil/config.go [hunk 1]
```diff
@@ -153,7 +150,7 @@ func filterCandidatesFromDirs(fname string, dirs []string) ([]string, error) {
 	candidate := getCandidate(fname, dirs)
 	fmt.Fprintf(os.Stderr, "candidate: %s\n", candidate)
 	if candidate == "" {
-		return nil, fmt.Errorf("file %s not found in %s", fname, dirs)
+		return nil, os.ErrNotExist
```
Comment (evelynl94) [inline utils/configutil/config.go]: Maybe we should define a different error type here. os.ErrNotExist is just for file not exist i think?

Comment (yiranwang52) [reply utils/configutil/config.go]: I think this is exactly what ErrNotExist is intended for.

Comment (yiranwang52) [reply utils/configutil/config.go]: This package needs to wrap the errors better. I will work on that.

Comment (evelynl94) [reply utils/configutil/config.go]: There is also this `ErrNoFilesToLoad` and not used anywhere...maybe we can remove that can let getCandidate returns any os errors.

Comment (yiranwang52) [reply utils/configutil/config.go]: Let me switch to ErrNoFilesToLoad

**Pull request merged**